client/driver/rkt.go

```diff
@@ -781,6 +789,13 @@ func (h *rktHandle) run() {
 	}
 	h.pluginClient.Kill()
 
+	// Remove the pod
+	if err := rktRemove(h.uuid); err != nil {
```

Comment:
Thinking: Should it use the created resources feature so that cleanup gets retried?

Reply:
I just tried to match Docker's behavior and this is how we do it there. I'll update rkt to use created resources.

Reply:
Investigated this route and it's a bit more work than I'd like to put in this PR. It involves returning CreatedResources from Driver.Start which isn't already happening and ensuring TaskDriver cleans up containers exactly when desired. Following Dockers lead for now seems safest.